### ISSUE TITLE
[CoE Starter Kit - BUG] Make mails more customizable by figuring out how to make the hard coded values also configurable.

### ISSUE THREAD

**Issue by poppynka** [microsoft/coe-starter-kit#2054]
### Describe the issue

Hi,

We are customizing the Welcome email good job with all the environment variables, it's so cool! We have the January release now and hope for this feature also for Governance solution 👍 

But to my problem - I am customizing the welcome email and expect it to add all e-mail parts from the variables, plus email body from the DV record. This all works great, BUT the action calling the email helper also adds some more HTML tags, which basically break my functionality in our customized environment variable _eMail Body Stop_

![image](https://user-images.githubusercontent.com/6829792/154072719-6b37af63-8caa-4c01-87c7-bac8efd79708.png)

thanks to these few lines we still need to keep the email in customizations solution, because we need to remove these lines.
I understand why these lines are there, because they are using environment variable to a community. We have rather included it to all emails in form of footer which is in stop body variable. It has to be hardcoded there, but it's in all emails now.

### Expected Behavior

if there is a functionality that provides this level of scalability to save us customizations, it should not include invading tags like these 🙂 

Or, there could be an id or class that I could use in case I need to hide this part 🙂 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.20.6

### What app or flow are you having the issue with?

Admin | Welcome Email v3

### Steps To Reproduce

1. customize eMail Stop Body
2. invoke sending Welcome email

### Anything else?

I know that this is just a customization and might not be for everyone (I actually think that most of the companies customize this email anyway 😄 ), but thanks for at least considering

AB#1255

**Comment by Jenefer-Monroe** [comment]
Is this a problem specific to this case in that it uses the <li> tag? Thus assumes that there is a list of objects above?

**Comment by poppynka** [comment]
yes, but we have customized the text of the email and we don't have there the list anymore. It's adding li and ending non-existing ul. This link to "Our Power Platform Community" now appears in our case on the top of the email above the logo 😄 

I have also removed div tags from the start and end of the body, because some email clients have problem to read divs and the layout was totally broken 🙄  I was sometimes very surprised what emails users really get

**Comment by poppynka** [comment]
but the div tag is of course in environment variables, this part works ok 🙂  it's jut unexpected list tags

**Comment by poppynka** [comment]
Moreover, when I try to customize the Admin | Welcome email, to remove the <li> and stuff, I can't simply create a copy of the flow and put it to customizations. It's using child flow, what makes it a bit more challenging. 

1. I need to remove the action calling child flow, otherwise I can't save a copy of the parent
2. Then I need to put the copy to the solution with customizations 
3. return the action calling the child without the "text" that I don't need (the <li> part)

Besides this, I really think that the functionality is awesome 🤩 
It's just adding any tags or text to what was retrieved from DV table, what makes it challenging to customize. 
Ideally, I would just take environment variables + the email body from DV and be able to construct a complete email in a separate .html file to test and see, how it looks like 🙂 These hardcoded lines are not ideal

**Comment by poppynka** [comment]
maybe if the hardcoded parts had something like id or class e.g. class="coe-kit-hardcoded" id="welcome-email". 
We could then simply `display: none` them in the eMail Header Style variable

**Comment by JanetK12** [comment]
Hello! Could I ask whether there are some new fixes regarding this issue/ bug?

**Comment by Jenefer-Monroe** [comment]
No I'm afraid we have not found a solution to this problem. We are open to ideas from you all though.

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.
